src/template.html

```diff
@@ -20,124 +20,122 @@
-            </p>
-            <p class="link">
-              <a href="#"
-                id="clone-insiders">Clone with VS Code Insiders</a>
-            </p>
-          </div>
-        </div>
-      
-        <div class="column git">
-          <div class="icon">
-            <img src="./images/git-logo.svg" alt="Git">
-          </div>
-          <div class="text">
-            <p class="title">Clone or push with Git</p>
-            <p class="description">Use git command line and your favorite editor to make changes and deploy to Azure.</p>
-            <p class="link">
-              <a href="#" id="show-creds">Generate your Git credentials</a>
-            </p>
-            <p id="creds" style="display: none"></p>
-      
-          </div>
-        </div>
-      </div>
-    </div>
+
 
     <div class="content-wrapper">
       <div class="content">
         <div class="column">
-          <div>
-              <p>2. Make a text change in <b>template.html</b></p>
+          <div class="content">
+
+            <div class="numberCircle">1</div>
+
+            <div>
+              <p class="title">Get the code</p>
+              <p> Click the links below to clone the sources for this application directly using Visual Studio Code, or
+                get git credentials to clone from the command line.</p>
+
+              <div>
+                <div class="icon">
+                  <img src="./images/vsc-logo.svg" alt="Visual Studio Code">
+                </div>
+                <p class="link">
+                  <a href="#" id="clone">Clone with Visual Studio Code</a>
+                </p>
+              </div>
+
+              <div>
+                <div class="icon">
+                  <img src="./images/BrandVisualStudioCode2017RC_256x.svg" alt="Visual Studio Code">
+                </div>
+                <p class="link">
+                  <a href="#" id="clone-insiders">Clone with Visual Studio Code Insiders</a>
+                </p>
+              </div>
+
+              <div>
+                <div class="icon">
+                  <img src="./images/git-logo.svg" alt="VS Code">
+                </div>
+                <p class="link">
+                  <a href="#" id="show-creds">Generate your Git credentials</a>
+                </p>
+                <p id="creds" style="display: none"></p>
+              </div>
+
+
+            </div>
+
           </div>
-        </div>
-      
-        <div class="column2">
-          <img src="./images/edit-screenshot.png" alt="VS Code edits">
-        </div>
-      </div>
-    </div>
+          <div class="content">
 
-    <div class="content-wrapper-light">
-      <div class="content">
-        <div class="column">
-          <div>
-            <p>3. Push with git to redeploy the app using the commands to the right.</p>
+            <div class="numberCircle-Light"></div>
             
+            <div>
+              <p class="title">Get the walkthrough</p>
```

Comment:
This is out of place with the rest of content. Any reason why this wasn't centered and placed below the content?